wfe2/wfe.go

```diff
@@ -2408,7 +2409,8 @@ func (wfe *WebFrontEndImpl) NewOrder(
 				return
 			}
 		} else {
-			logEvent.IgnoredRateLimitError = err.Error()
+			wfe.sendError(response, logEvent, web.ProblemDetailsForError(err, "While checking rate limits"), err)
```

Comment:
I think this needs to be gated on `if features.Get().UseKvLimitsForNewOrder {`

```suggestion
			wfe.sendError(response, logEvent, web.ProblemDetailsForError(err, "While checking rate limits"), err)
```

Reply:
Good catch, thanks! 🔍